tests/spec/fs.stat.spec.js

```diff
@@ -91,4 +91,33 @@ describe('fs.stat', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
+    var fs = util.fs();
```

Comment:
Maybe consider doing this to make it more clear what is being tested.
var fsPromises = util.fs().promises;

Reply:
Could you please elaborate on the necessity of this change?

Reply:
As the rest of the tests in this file are for fs, it's just better to make it more obvious you're testing for fsPromises.